pydicom/config.py

```diff
@@ -50,6 +84,11 @@ def DS_decimal(use_Decimal_boolean=True):
     use_Decimal_boolean : bool
         ``True`` to derive :class:`~pydicom.valuerep.DS` from
         :class:`decimal.Decimal`, ``False`` to derive it from :class:`float`.
+
+    Raises
+    ------
+    ValueError
+        If use_Decimal_boolean and use_DS_numpy are both True.
```

Comment:
If `use_Decimal_boolean` and `config.use_DS_numpy` are both ``True``. Or a reference to use_DS_numpy if available